src/main/java/entity/CarColor.java

```diff
@@ -0,0 +1,8 @@
+package entity;
+
+public enum CarColor {
```

Comment:
In my opinion it's tight Java code with database too much. So better to send this logic to db.